### DISCUSSION TITLE
Can a github page made using default templates ALSO render a custom html file in the same repository

### DISCUSSION THREAD

**Discussion by armsp** [community/community#22981] [Other Feature Feedback, Questions, & Ideas]
<p>I have setup a Github page successfully using Jekyll and the default Minimal theme on the  <strong>gh-pages</strong>  branch. It acts as a nice documentation for my code in the  <strong>master </strong> branch. However I now want it to also render a  <strong>custom html page</strong>.</p>
<p><strong>What do I mean by that?</strong></p>
<p>Usually we write in markdown and Github takes care of everything (when using the Jekyll themes). But this custom html file can’t be prepared that way. It has some interactive elements, scripts etc that I don’t know how to put down in markdown and expect to be prepared correctly by Github (OR is it actually possible to hack the template to do that? ). I don’t really care the endpoint of the page, it can be any thing.</p>
<p>So finally, I want to have the github project page with say Minimal theme working with the usual markdown files and that only one or more files would be in HTML directly and in the same “<a href="http://username.github.io/repo-name/xxxxx" class="inline-onebox-loading" rel="nofollow noopener">username.github.io/repo-name/xxxxx</a>” domain as the usual markdown files appear.</p>
<p>If it can be done, please let me know how.</p>
<p>I feel like it should be doable but perhaps the way I am googling isn’t yielding anything useful.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Yes, you can find an example in <a href="https://www.lee-dohm.com/pages-test/" rel="nofollow noopener">this Pages test repository I created</a>. You can see <a href="https://github.com/lee-dohm/pages-test" rel="nofollow noopener">the contents of the repository here</a>. For more information about the capabilities of Jekyll, see <a href="https://jekyllrb.com" rel="nofollow noopener">the Jekyll documentation site</a>.</p>
<p>Let us know if you have more questions.</p>
